doc/examples/uuid.rst

```diff
@@ -255,19 +255,27 @@ Applications can set the UUID representation in one of the following ways:
       * - ``unspecified``
         - :ref:`unspecified-representation-details`
 
-#. Using the ``uuid_representation`` kwarg option, e.g.::
+#. At the ``MongoClient`` level using the ``uuidRepresentation`` kwarg
+   option, e.g.::
 
      from bson.binary import UuidRepresentation
-     client = MongoClient(uuid_representation=UuidRepresentation.PYTHON_LEGACY)
+     client = MongoClient(uuidRepresentation=UuidRepresentation.PYTHON_LEGACY)
 
-#. By supplying a suitable :class:`~bson.codec_options.CodecOptions`
-   instance, e.g.::
+#. At the ``MongoClient`` level by supplying a suitable
```

Comment:
MongoClient -> Database or Collection level?

This isn't changing the option on the MongoClient level.

Reply:
Ah, right. Updated.